lua/nvim-treesitter/parsers.lua

```diff
@@ -391,6 +391,16 @@ list.glimmer = {
   filetype = "handlebars",
 }
 
+list.pug = {
+  install_info = {
+    url = "https://github.com/zealot128/tree-sitter-pug",
+    files = { "src/parser.c", "src/scanner.cc" },
+  },
+  maintainers = { "@zealot128" },
+  filetype = "pug",
+  used_by = {"vue"}
```

Comment:
Why `used_by` vue? This isn't necessary for language injection and would only have an effect if a vue parser wouldn't exist and the filetype should use the pug parser.

Reply:
removed it! as my prime goal was to get the pug highlighting within vue working I tried all avenues I could find :) In the end it is necessary to change the parser grammar within tree-sitter-vue to make it work, which I also submitted a [PR](https://github.com/ikatyang/tree-sitter-vue/pull/13). 
I removed the used_by line.